lib/dash/segment_template.js

```diff
@@ -174,6 +174,7 @@ shaka.dash.SegmentTemplate.parseSegmentTemplateInfo_ = function(context) {
     timescale: segmentInfo.timescale,
     startNumber: segmentInfo.startNumber,
     presentationTimeOffset: segmentInfo.presentationTimeOffset,
+    unscaledPresentationTimeOffset: segmentInfo.unscaledPresentationTimeOffset,
```

Comment:
Add this field to the `shaka.dash.SegmentTemplate.SegmentTemplateInfo` definition.